doc/releasenotes.html

```diff
@@ -48,17 +54,205 @@ <h1 id="yyyy-mm-dd-r[n]">SAM yyyy.m.d [r[n]], SSC nnn: Month d, yyyy</h1>
+
+<h2>Financial Models (SAM 2024.12.12)</h2>
+
+<p>Update investment-tax credit (ITC) and depreciation basis calculations to remove financing fees and reserve account funding from basis. <a href="https://github.com/NREL/SAM/pull/1860">SAM 1860</a>, <a href="https://github.com/NREL/ssc/pull/1193">SSC 1193</a>)</p>
+
+<p>Add option for DSCR debt sizing option (debt-service coverage ration as input, or sculpted debt, sizing based on cash available for debt service) that was available only for Single Owner financial model to other front-of-meter financial models. (<a href="https://github.com/NREL/SAM/pull/1854">SAM 1854</a>, <a href="https://github.com/NREL/ssc/pull/1208">SSC 1208</a>)</p>
+
+
+<p>Add internal rate of return (IRR) metric to Residential and Commercial financial models. Note that the value of these projects is based on electricitiy bill savings rather than revenue. (<a href="https://github.com/NREL/ssc/pull/1223">SSC 1223</a>)</p>
+
+<p>Add nominal discount rate output variable to financial models to help with analyses involving net present value (NPV). (<a href="https://github.com/NREL/ssc/pull/1223">SSC 1223</a>)</p>
+
+<h2>Electricity Bill Calculator (SAM 2024.12.12)</h2>
+
+<p>Add option for net billing with carryover for credits that expire at end of true-up period instead of carrying over to following year, and credits apply to energy charge portion of monthly electricity bill. (<a href="https://github.com/NREL/SAM/pull/1835">SAM 1835</a>, <a href="https://github.com/NREL/ssc/pull/1200">SSC 1200</a>)</p>
+
+<p>Improve error checking of time-of-use (TOU) rates by requiring that tiers are the same within a month across TOU periods. (<a href="https://github.com/NREL/ssc/pull/1207">SSC 1207</a>)</p>
+
+<h2>Detailed PV (SAM 2024.12.12)</h2>
+
+<p>Improve solar position algorithm to speed up PVWatts and Detailed PV simulation run times. (<a href="https://github.com/NREL/ssc/pull/1212">SSC 1212</a>)</p>
+
+<p>Update module and inverter libraries. (<a href="https://github.com/NREL/SAM/pull/1913">SAM 1913</a>, <a href="https://github.com/NREL/SAM/pull/1917">SAM 1917</a>)</p>
+
+<p>Rename temperature coefficient of power variable from 'gamma_r' to 'gamma_pmp' to be more consistent with industry standard. (<a href="https://github.com/NREL/ssc/pull/1231">SSC 1231</a>, <a href="https://github.com/NREL/SAM/pull/1898">SAM 1898</a>)</p>
+
+<p>Add subhourly clipping correction option based on matrix lookup approach ("Allen Method"). (<a href="https://github.com/NREL/SAM/pull/1861">SAM 1861</a>)</p>
+
+<p>Fix six-parameter module coefficient generator sanity check that caused a crash. This should result in more modules from the CEC list to be inlcuded in the SAM module library. (<a href="https://github.com/NREL/ssc/pull/1245">SSC 1245</a>)</p>
+
+<p>Fix solar position algorithm issue that caused SAM to crash when running multi-threaded such as for parametric simulations. (<a href="https://github.com/NREL/ssc/pull/1232">SSC 1232</a>)</p>
```

Comment:
I believe this is a dev bug, as the issue with multi-threading only happened once the hash table fix was merged in.